pkg/payload/payload.go

```diff
@@ -213,7 +221,7 @@ func LoadUpdate(dir, releaseImage, excludeIdentifier, profile string) (*Update,
 	return payload, nil
 }
 
-func shouldExclude(excludeIdentifier, profile string, manifest *manifest.Manifest) bool {
+func shouldExclude(excludeIdentifier, profile string, manifest *manifest.Manifest, includeTechPreview bool) bool {
```

Comment:
nit: this is asking about manifest exclusion based on some config knobs. I'm agnostic about whether the manifest argument goes before or after those knobs, but it probably shouldn't sit in the middle of the knobs.